generator/testdata/rust/protobuf/golden/module/rpc/mod.rs

```diff
@@ -235,8 +235,8 @@ pub mod quota_failure {
     pub struct Violation {
 
         /// The subject on which the quota check failed.
-        /// For example, "clientip:<ip address of client>" or "project:<Google
-        /// developer project id>".
+        /// For example, "clientip:&lt;ip address of client&gt;" or "project:&lt;Google
```

Comment:
Can you use `\<` instead of `&lt;` ?  No big deal if you cannot, I just find the backslash escaping more readable, but both render the same I assume.